taskmgr/server/server_test.go

```diff
@@ -39,3 +42,58 @@ func TestGetTasksHandler(t *testing.T) {
 	})
 
 }
+
+func TestNewTaskHandler(t *testing.T) {
+
+	var IMD = inMemoryDatabase{
+		tasks: []*task.Task{},
+	}
+
+	newDummyTask := struct {
+		Title string
+		Desc  string
+	}{
+		Title: "new upcomig task",
+		Desc:  "title of a new fancy task",
+	}
+
+	t.Run("check new task created", func(t *testing.T) {
+
+		jsonTask, err := json.Marshal(newDummyTask)
+		if err != nil {
+			t.Errorf("Unable to marshal the dummy task")
+			return
```

Comment:
here you can use t.Fatalf to immediately stop the program no need for a return fatalf throws a panic and exits
```suggestion
			t.Fatalf("Unable to marshal the dummy task")
```